src/main/ipc.ts

```diff
@@ -106,6 +161,7 @@ function configureProtocol(options: ElectronMainOptionsInternal): void {
 function configureClassic(options: ElectronMainOptionsInternal): void {
   ipcMain.on(IPCChannel.EVENT, ({ sender }, jsonEvent: string) => handleEvent(options, jsonEvent, sender));
   ipcMain.on(IPCChannel.SCOPE, (_, jsonScope: string) => handleScope(options, jsonScope));
+  ipcMain.on(IPCChannel.ENVELOPE, ({ sender }, env: Uint8Array | string) => handleEnvelope(options, env, sender));
```

Comment:
Is there a possibility that users will listen on the `IPCChannel.EVENT` themselves to do some stuff but since the `EventToMain` integration is gone by default the channel will not receive anything anymore?

I think I am fine with that being the case, however we should probably mention this in the changelog somewhere.

Reply:
We don't export `IPCChannel` so to listen for this event they'd need find and use the correct string (`sentry-electron.event`). 

I suppose our IPC usage is an internal, undocumented API.

Reply:
Ok right. Then it probably doesn't matter at all.